doc/GeneratingDocumentation.md

```diff
@@ -52,3 +52,29 @@ mkdir -p target/documentation
 avrodoc target/schemas/reads.avpr > target/documentation/reads.html
 
 ```
+
+### Automating the process
+
+Once you have installed Avrodoc, you can run the `contrib/make_avrodoc.sh` script to automate the above process, building Avrodoc HTML files for each `.avdl` in `target/documentation`:
+
+```shell
+contrib/make_avrodoc.sh
+```
+
+## Generating UML Diagrams
+
+There is a UML class diagram, `doc/uml.dia`, that describes the layout of the GA4GH data model.
+
+UML class diagrams can be partially generated from the schemas by using [Doxygen](http://www.doxygen.org/) to generate XML, and then using [Dia](http://live.gnome.org/Dia) to generate a UML from that XML output. Unfortunately, this only imports the Avro types: dependencies and layout still need to be done manually.
```

Comment:
Nah, that wasn't my intention. I think it makes perfect sense to include both the source file and the resulting SVG, as you did here. My only suggestion (perhaps better aimed at the CONTRIBUTING.md, now that I think about it) was to allow for optional future inclusion of .vsdx files if the creator had used a tool that natively supports them.